recipe/meta.yaml

```diff
@@ -8,14 +8,14 @@ source:
     md5: f857f49a66918316996eb52399e04120
 
 build:
-    number: 3
+    number: 4
     skip: True  # [win]
 
 requirements:
     build:
-        - libpng
-        - zlib
-        - jpeg 9b
+        - libpng 1.6*
+        - zlib 1.2*
+        - jpeg 9*
         - jasper
```

Comment:
@conda-forge/g2clib - is there an ABI dependency on jasper? How deep and which version? i.e. is it just the major version, major + minor, or major + minor + patch?